docs/aro/setup-quay/quay-console.md

```diff
@@ -0,0 +1,316 @@
+![Quay Logo](./images/redhat-quay-logo.png)
+
+## Red Hat Quay setup on ARO (Azure Openshift)
+A Quickstart guide to deploying an Azure Red Hat OpenShift Cluster with Red Hat Quay.
+
+Author: [Kristopher White x Connor Wooley]
+
+## Video Walkthrough
+
+If you prefer a more visual medium, you can watch [Kristopher White] walk through Quay Registry Storage Setup on [YouTube]([https://youtu.be/iifsB-uuEFc](https://youtu.be/yMmSrx4hN70)).
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/yMmSrx4hN70" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+
+## Prerequisites
+
+### Azure CLI
```

Comment:
no need for these instructions, refer back to ARO quickstart guide, or the CLI install docs.